src/bridge/client/client.rs

```diff
@@ -612,6 +612,65 @@ impl BitVMClient {
         }
     }
 
+    pub async fn process_peg_in_as_depositor(&mut self,peg_in_graph: &PegInGraph) {
```

Comment:
This function and `process_peg_in_as_verifier()` function belong to client_command.rs file as we handle user's commands there. Or we could create such functions for all client roles in both peg-in and peg-out graphs and use them in `handle_automatic_command()` function.

Reply:
Please see the comment above